response_operations_ui/templates/edit-survey-details.html

```diff
@@ -96,15 +96,18 @@ <h1 class="ons-u-fs-xl">Edit survey details</h1>
                         })
                     }}
                 {% endcall %}
-
-                {{
-                    onsButton({
-                        "classes": "ons-u-mb-s ons-u-mt-l",
-                        "text": "Save",
-                        "id": "save-btn"
-                    })
-                }}
-                <p><a id="cancel-btn" href="{{ url_for('surveys_bp.view_surveys') }}">Cancel</a></p>
+                <div class="ons-grid ons-grid--flex ons-grid--gutterless ons-grid--vertical-center ons-grid--no-wrap@s ons-u-mt-m ons-u-mb-m">
```

Comment:
Are we sure we need so many classes attached to a button?

Reply:
This fix was to use the grid system to get the save/submit button and cancel link to be displayed alongside each other. The classes are all needed for this might be able to remove `ons-u-mb-m` for the bottom margin if needed though